class.jetpack.php

```diff
@@ -669,6 +670,14 @@ function point_edit_links_to_calypso( $default_url, $post_id ) {
 		return esc_url( sprintf( 'https://wordpress.com/%s/%s/%d', $path_prefix, $site_slug, $post_id ) );
 	}
 
+	function point_edit_comment_links_to_calypso( $url ) {
```

Comment:
Any reason why the signature is different for this function versus `point_edit_post_links_to_calypso`? I think it's clearer to pass through a fallback and a comment id explicitly.

Reply:
`point_edit_post_links_to_calypso` is hooked to `get_edit_post_link`, which [passes in](https://developer.wordpress.org/reference/hooks/get_edit_post_link/) the original link, the post ID (very convenient for our needs), and a context (not needed in our situation, so [we only grab the first two](https://github.com/Automattic/jetpack/blob/master/class.jetpack.php#L637) ).

The `get_edit_comment_link` filter, however, [only gives us the link we'll be filtering](https://developer.wordpress.org/reference/hooks/get_edit_comment_link/), hence all the parsing madness.

Reply:
👍 Got it thanks for clarifying!